_data/governance.yml

```diff
@@ -0,0 +1,61 @@
+    links:
+      - text: "This process aims to evolve Scala openly and collaboratively. Anyone from the community is welcome to submit a Scala Improvement Proposal (SIP), which is then reviewed and discussed by a Committee. Every month, the Committee votes on the proposals to accept in the language."
+      - name: SIP Homepage
+        url: https://docs.scala-lang.org/sips/index.html
+      - name: SIP Committee Members
+        url: https://docs.scala-lang.org/sips/process-specification.html#the-sip-committee
+      - name: SIP Process Specification
+        url: https://docs.scala-lang.org/sips/process-specification.html
+  - title: "Scala 3 Maintenance"
+    description: "Managing the Scala 3 compiler, standard library and documentation."
+    expandText: "Process & People"
+    links:
+      - name: Core Maintainers
+        url: https://github.com/lampepfl/dotty/blob/main/MAINTENANCE.md#compiler
+      - name: Maintenance Procedures
+        url: https://github.com/lampepfl/dotty/blob/main/MAINTENANCE.md
+  - title: "Tooling"
+    description: "Managing build tools, linters, IDEs, and other tools."
+    expandText: "Process & People"
+    links:
+      - text: "Process and People are TBD, please stay tuned! However you can read below about the Scala Tooling Summit:"
+      - name: "Scala Tooling Summit of March 2023"
+        url: https://www.scala-lang.org/blog/2023/04/11/march-2023-scala-tooling-summit.html
+  - title: "Moderation"
+    description: "Learn how the Scala community is moderated."
+    expandText: "Process & People"
+    links:
+      - text: "The Scala community is moderated by the Moderation Team over all the official communication channels. The moderation is governed by the Code of Conduct"
+      - name: "Moderation Team"
+        url: https://www.scala-lang.org/conduct/#contact
+      - name: Code of Conduct
+        url: https://www.scala-lang.org/conduct/
+      - name: "Inclusive Language Guide"
+        url: https://docs.scala-lang.org/contribute/inclusive-language-guide.html
+      - name: "Communication Channels"
+        url: https://docs.scala-lang.org/community/
+  - title: "Scala Center"
+    description: "The organization fostering the Scala community, education, and OSS library development."
+    expandText: "Learn More"
+    links:
+      - name: "Homepage"
+        url: https://scala.epfl.ch/
+      - name: Team
+        url: https://scala.epfl.ch/team.html
+      - name: "Advisory Board"
+        url: https://github.com/scalacenter/advisoryboard#the-scala-center-advisory-board
+      - name: "5 Year Report"
+        url: https://scala.epfl.ch/records/first-five-years/
+  - title: "Scala 2 Maintenance"
+    description: "Who maintains the Scala 2 compiler?"
+    expandText: "Process & People"
+    links:
+      - name: "Contribution Process"
+        url: "https://github.com/scala/scala#how-to-contribute"
+      - name: "Core Maintainers"
+        url: "https://github.com/scala/scala#get-in-touch"
```

Comment:
Thank you for adding this card. I am a bit confused because its content is pretty different from the Scala 3 Maintenance card. Would it be possible to make them more similar?

Reply:
This would require making the actual processes similar :) which is not the case.

In the case of Scala 3, we have an Issue Supervisor role, which is specced as an actual procedure that we follow weekly to monitor the new issues, identify critical ones, and schedule them in a milestone for fixing. For Scala 2, the link I specified doesn't describe a maintenance procedure but rather how to push your own contribution.

Since the nature of the two documents is so different, I don't think we can't label the Scala 2 link as Maintenance Procedures in the same way the Scala 3 one is called that way.